code/modules/surgery/organs/helpers.dm

```diff
@@ -117,7 +117,7 @@
 	var/obj/item/tank/jetpack/J
 	if(istype(back,/obj/item/tank/jetpack))
 		J = back
-		if(J.on == 1)
+		if(J.on)
```

Comment:
This whole `get_leg_ignore` proc (and `get_missing_limbs` below) is absolutely unused, but  removing them is technically out of scope of the PR